pipeline.config

```diff
@@ -25,6 +25,7 @@ CI:
   # added to allow mixed case branch names (toolchains fail on mixed case branch names without this set,
   # could remove when we switch to using main branch
   RELEASE_NAME_NO_GIT_BRANCH: "true"
+  USEHADOLINT: "true"
```

Comment:
Just curious as to why you enabled hadolint? I'm not familiar with it's advantages/disadvantages dockerlint.

Reply:
Looks like it was possibly to add support  for inline comments within commands

Reply:
I believe that's right.
[Here's](https://github.ibm.com/watson-health-cohorting/spark-history-server-spike/blob/master/Dockerfile#L14) and example of what he's talking about.
I believe the normal Dockerlint blows up on that particular case.